glossary.yml

```diff
@@ -1606,6 +1606,13 @@
     term: ወደኋላ-ተኳሃኝ
     def: >
       እርስ በእርስ ለመተባበር የሚያስችል የአንድ ስርዓት ፣ ሃርድዌር ወይም ሶፍትዌር ንብረት በድሮው የቅርስ ስርዓት ወይም ለእንዲህ ዓይነቱ ስርዓት በተዘጋጀ ግብዓት። ለምሳሌ በ [ፓይዘን](#function) 3 ውስጥ የተፃፈ [ተግባር](#python) 3 ሊሆን ይችላል ከፓይዘን ስሪት 2 ጋር በተሳካ ሁኔታ ያሂዱ ከኋላ-ተኳሃኝ ነው። የአንድ ስርዓት ፣ ሃርድዌር ወይም ሶፍትዌር ንብረት በድሮው የቅርስ ስርዓት ወይም ለእንዲህ ዓይነቱ ስርዓት  እርስ በእርስ ለመተባበር እንዲያስችል የተዘጋጀ ግብዓት። ለምሳሌ [ተግባር](#python)  በ [ፓይዘን](#function) 3 ውስጥ የተፃፈ ከፓይዘን ስሪት 2 ጋር በተሳካ ሁኔታ የሚሄዱ ከኋላ-ተስማሚ ነው።
+  uk:
+    term: "зворотна сумісність"
+    def: >
+      Властивість системи, апаратного або програмного забезпечення, що дозволяє взаємодіяти
+      зі старішою спадковою системою або з вхідними даними, призначеними для такої системи.
```

Comment:
"спадкова" чи "успадкована" чи мабуть щось інше - варто пошукати в літературі

Reply:
https://slovnyk.ua/index.php?swrd=%D1%83%D1%81%D0%BF%D0%B0%D0%B4%D0%BA%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B9

Всеж таки `успадкованої системи`

Reply:
@Dod900ls1 А чи слово "успадкованою" використовується щодо програмного забезпечення, яке було створено у далекому минулому?

Reply:
I can't find any data that would approve or disapprove this. Shall we just get rid of this problematic word?

Reply:
Maybe. Although it may be nice to translate "legacy system" to Ukrainian too, maybe in a separate article.

I do have indeed some reservations against saying "with an older legacy system" since it does not have to be a legacy one. E.g. I tend to understand legacy code as something that no longer supported, or is very much outdated etc. But if I just made a new release X.Y+1 of my software, and added a new optional keyword argument for a Python function A, it will be backwards compatible with code written for version X.Y not using that keyword argument. That does not make version X.Y a legacy one, I think.

Maybe this something to consider for English version too (CC @elletjies and @froggleston)? E.g. saying "A property of a system, hardware or software, that allows for interoperability with an older system" without the word "legacy", or "... with an older version of the system or even another legacy system".

Reply:
@olexandr-konovalov @Herobread I removed the word "успадкованою". Please review.

Reply:
I think it's OK now, especially with the explanation for Python as an example. @Herobread ?